content/hi/docs/reference/glossary/container-env-variables.md

```diff
@@ -0,0 +1,16 @@
+---
+title: कंटेनर पर्यावरण चर (Container Environment Variables)
+id: container-env-variables
+date: 2018-04-12
+full_link: /docs/concepts/containers/container-environment/
+short_description: >
+  कंटेनर वातावरणीय वेरिएबल्स नाम=मान जोड़े होते हैं जो पॉड में चल रहे कंटेनरों को उपयोगी जानकारी प्रदान करते हैं
```

Comment:
Instead of `वातावरणीय` use `एनवायरनमेंट`  everywhere in the entire docs here

Reply:
The sentence will look more good like this 
कंटेनर एनवायरनमेंट वेरिएबल्स नाम=मान (name=value) की जोड़ी होते हैं जो पोड में चल रहे कंटेनरों को उपयोगी जानकारी प्रदान करते हैं।